pkg/storage/wal/ref.go

```diff
@@ -0,0 +1,13 @@
+package wal
+
+type DataRef uint64
+
+func NewDataRef(offset, size uint64) DataRef {
+	return DataRef(offset<<32 | size)
+}
+
+func (b DataRef) Unpack() (int, int) {
```

Comment:
Whats the reason to pack these values?

Reply:
None really, we could use 2 int64 or 2 int32. The code was just available and I copy pasted.

Do you prefer to change it ?

Reply:
If we're using it elsewhere then I'm happy to continue using it. If this is the first usage I'm less inclined to introduce this as a way of storing multiple values unless there is a good reason (maybe due to perf, storage concerns etc.)